### PR TITLE
Smb2 version

### PR DESCRIPTION
**Pull request by ethergrouik**
My first nse script for retrieving the operating system version of a Windows station using the SMB2 protocol.

$ ./nmap -Pn -n -p 445 --script=smb2-version 192.168.56.130
Starting Nmap 7.80SVN ( https://nmap.org ) at 2020-03-04 11:55 CET
Nmap scan report for 192.168.56.130
Host is up (0.00020s latency).

PORT    STATE SERVICE
445/tcp open  microsoft-ds

Host script results:
| smb2-version: 
|   OS: Windows Server 2016, Version 1607
|   Server NetBIOS domain name: TEST
|   Server NetBIOS computer name: AD2016
|   DNS domain name: test.local
|   FQDN: AD2016.test.local
|   DNS forest name: test.local
|_  Date: 2020-03-04T19:55:40

Nmap done: 1 IP address (1 host up) scanned in 0.13 seconds

### CODE REVIEW

**Comment by ethergrouik** [comment]
Hello,
Could anyone tell me if there is a problem with my script?
It would be really nice of you to give me some feedback.
Thanks everyone!

File: scripts/script.db [hunk 1]
```diff
@@ -499,7 +499,9 @@ Entry { filename = "smb-webexec-exploit.nse", categories = { "exploit", "intrusi
 Entry { filename = "smb2-capabilities.nse", categories = { "discovery", "safe", } }
 Entry { filename = "smb2-security-mode.nse", categories = { "default", "discovery", "safe", } }
 Entry { filename = "smb2-time.nse", categories = { "default", "discovery", "safe", } }
+Entry { filename = "smb2-version.nse", categories = { "default", "discovery", "safe", "version", } }
 Entry { filename = "smb2-vuln-uptime.nse", categories = { "safe", "vuln", } }
+Entry { filename = "smb3-vuln-cve2020-0796.nse", categories = { "dos", "exploit", "intrusive", "vuln", } }
```
Comment (cnotin) [inline scripts/script.db:504]: This script isn't in this PR

**Comment by cnotin** [comment]
Caution: if we combine the changes by all your commits we obtain two "smb2-version.nse" files, one at root folder and one in scripts/
They are different too ;)
You can rebase your PR and merge all of these so it'll be clearer :)

**Comment by ethergrouik** [comment]
Thank You cnotin for your comments.

**Pull request closed without merging**